starter-test/pom.xml

```diff
@@ -0,0 +1,68 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <artifactId>operator-framework-spring-boot-starter-parent</artifactId>
+    <groupId>io.javaoperatorsdk</groupId>
+    <version>1.9.0-SNAPSHOT</version>
+    <relativePath>..</relativePath>
+  </parent>
+  
+  <name>Java Operator SDK Spring Boot Starter Test</name>
+  <description>Spring Boot starter for the Java Operator SDK</description>
```

Comment:
I think you need to adjust the description here to mention that this is the test starter…

Reply:
Yes indeed!